### PR TITLE
VCDL-130 getting rid of deprecatd GitHub parameters

### PR DESCRIPTION
**Pull request by jennamwong**

### CODE REVIEW

File: scripts/configure_git.bash
Comment (thomashashi) [inline scripts/configure_git.bash:1]: Only one comment on this file: every place you commented out the deprecated stuff, just delete it. We have the history of why that is no longer in this file in git, no need to keep around stuff which we don't want to run.

File: terraform-cloud/exercises/08-variables.tf [hunk 1]
```diff
@@ -4,7 +4,8 @@ variable "oauth_connection_name" {
   default = "HashiCat Workshop"
 }
 
-variable "github_organization" {
+// previously github_organizaition but this argument is deprecated
```
Comment (thomashashi) [inline terraform-cloud/exercises/08-variables.tf:7]: This is fine however - you add a comment saying what this used to be but it's now this new thing.

File: scripts/configure_git.bash
Comment (thomashashi) [reply scripts/configure_git.bash:1]: Once you do this, feel free to merge.

**Comment by jennamwong** [comment]
Closing this to take out the comments Thomas recommended.

**Pull request closed without merging**